app/assets/stylesheets/homepage.css.scss

```diff
@@ -190,49 +200,48 @@ margin-left: 5%;
 }
 
 .button-text {
-    font-size: 36px;
+  font-size: 36px;
 }
 
 .plan-trip-question {
-    font-size: 24px;
-    font-weight: 700;
+  font-size: 24px;
+  font-weight: 700;
 }
 
 
 .home-mission {
-    text-align: center;
-    height: 400px;
-    margin-top: 8%;
+  text-align: center;
+  margin-top: 8%;
+  margin-bottom: 45px;
 }
 
 .home-mission p {
-    text-align: center;
-    font-size: 24px;
-    font-weight: 200;
-    padding: 75px;
-    color: white;
-    text-shadow: black 2px 2px 2px;
+  text-align: center;
```

Comment:
Properties should be ordered background, background-color, font-size, line-height, padding, text-align